src/components/Content.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { removeTodos, saveTodo } from '../api/todos';
+
+import { Todo } from '../types/Todo';
+import { Header } from './Header';
+import { Main } from './Main';
+import { Footer } from './Footer';
+import { TodoItem } from './TodoItem';
+
+import { Filter } from '../types/Filter';
+import { Error } from '../types/Error';
+
+type Props = {
+  todos: Todo[],
+  changeTodos: (todos: Todo[]) => void,
+  userId: number,
+  onError: (errorText: Error) => void,
+};
+
+export const Content: React.FC<Props> = ({
+  todos,
+  changeTodos,
+  userId,
+  onError,
+}) => {
+  const [filter, setFilter] = useState<Filter>(Filter.all);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [isAdded, setIsAdded] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+
+  const onChangeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364